app/views/equipment/show.html.erb

```diff
@@ -4,7 +4,7 @@
 
   reservations = []
   rev = @equipment.reservations
-  rev = rev.confirmed.or(rev.blocked)
+  rev = rev.confirmed.or(rev.blocked).or(rev.pending)
```

Comment:
I modified this code in #160, there will most likely be merge conflicts, let's keep an eye out for that.